common/rc

```diff
@@ -140,3 +143,101 @@ _get_pci_dev_from_blkdev() {
 	readlink -f "$TEST_DEV_SYSFS/device" | \
 		grep -Eo '[0-9a-f]{4}:[0-9a-f]{2}:[0-9a-f]{2}\.[0-9a-f]'
 }
+
+_notrun()
```

Comment:
Like I noted above, this won't work in blktests. The `_have_*` helpers are how you do this in blktests.